### DISCUSSION TITLE
Switch ports and traffic aggregation

### DISCUSSION THREAD

**Discussion by jfreak53** [netbox-community/netbox#12943] [Q&A] [answered]
I ran the demo, but have not installed it on my own server yet. Just trying to figure out if it will or not. Can NetBox integrate with my Mikrotik switches and put ports up/down? As well aggregate traffic for me?

I figured I can do the manual part of creating a module for WHMCS to connect it to my billing platform, but the rest I was curious about. Thanks for any direction!

**Comment by peterbaumert** [comment, marked as answer]
Netbox itself is only for storing information. It does not act on switches or anything. For this you need for example ansible or saltstack which can use netbox as a source of its information.
